assets/src/components/parts/janus-popup/Popup.tsx

```diff
@@ -51,8 +51,12 @@ const Popup: React.FC<PartComponentProps<PopupModel>> = (props) => {
     // result of init has a state snapshot with latest (init state applied)
     setInitSnapshot(initResult);
     const currentStateSnapshot = initResult.snapshot;
+    const isOpenByDefault: boolean | undefined = currentStateSnapshot[`stage.${id}.openByDefault`];
```

Comment:
For fixing one of the issue in Seasons lesson, I removed this one but we need this. Also, I have changed the sequence of these 2 variables i.e. isopen and openbydefault.